src/pages/VaccinePage.js

```diff
@@ -1,16 +1,104 @@
+/* eslint-disable react/forbid-prop-types */
 /**
  * mSupply Mobile
  * Sustainable Solutions (NZ) Ltd. 2016
  */
 
 import React from 'react';
-import { View } from 'react-native';
 import { connect } from 'react-redux';
+import { StyleSheet, Text, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
 
-export const VaccinePageComponent = () => <View />;
+import { FridgeDisplay, FlexView } from '../widgets';
 
-const mapStateToProps = state => state;
+import { selectMinAndMaxLogs, selectMinAndMaxDomains } from '../selectors/fridge';
+import { FridgeActions } from '../actions/FridgeActions';
 
-const mapDispatchToProps = dispatch => ({ dispatch });
+import { APP_FONT_FAMILY } from '../globalStyles';
+import { vaccineStrings } from '../localization';
+
+export const VaccinePageComponent = ({
+  selectedFridge,
+  minLine,
+  maxLine,
+  breaches,
+  maxDomain,
+  minDomain,
+  fridges,
+  onSelectFridge,
+}) => {
+  const Fridge = React.useCallback(
+    ({ item }) => (
+      <FridgeDisplay
+        fridge={item}
+        minLine={minLine}
+        maxLine={maxLine}
+        breaches={breaches}
+        maxDomain={maxDomain}
+        minDomain={minDomain}
+        onSelectFridge={onSelectFridge}
+        isActive={item.id === selectedFridge.id}
+      />
+    ),
+    minLine,
+    maxLine,
+    breaches,
+    maxDomain,
+    minDomain,
+    onSelectFridge
```

Comment:
Do we need to add `selectedFridge` to deps here?

Reply:
ya 2f5a123d7ee9f1f61f331d4497e08fc65737b5cd

(look at the commit, theres another funny bug there actually 🤣 )